experimental/Schemes/src/Resolution_structure.jl

```diff
@@ -364,10 +369,93 @@ function add_map_embedded!(f::BlowUpSequence, phi::BlowupMorphism)
   return f
 end
 
+function extend(f::MixedBlowUpSequence, g::Union{BlowUpSequence,MixedBlowUpSequence})
+  for g_i in morphisms(g)
+    add_map!(f,g_i)
+  end
+  return f
+end
+
+function _blow_up_at_all_points_embedded(f::BlowUpSequence, I_all::AbsIdealSheaf)
+  @assert domain(f) == scheme(I_all)
+
+  decomp=maximal_associated_points(I_all)
+  while length(decomp)>0
+    I = small_generating_set(pop!(decomp))
+    f = _do_blow_up_embedded!(f,I)
+    if length(decomp)>0
+      decomp = [strict_transform(last_map(f),J) for J in decomp]
+    end
+  end
+  return f
+end
+
+function _blow_up_at_all_points_embedded(f::BlowUpSequence, V::Vector{AbsIdealSheaf})
+   I = small_generating_set(pop!(V))
+   f = _do_blow_up_embedded!(f,I)
+   if length(V) > 0
+     tempV = [strict_transform(last_map(f),J) for J in decomp]
+     f = _blow_up_at_all_points_embedded(f,tempV)
+   end
+   return f
+end
+
+function _blow_up_at_all_points(f::Union{BlowUpSequence, MixedBlowUpSequence}, I_all::AbsIdealSheaf)
+  @assert domain(f) == scheme(I_all)
+
+  decomp=maximal_associated_point(I_all)
+  while length(decomp)>0
+    I = small_generating_set(pop!(decomp))
+    f = _do_blow_up!(f,I)
+    if length(decom)>0
+      decomp = [strict_transform(last_map(f),J) for J in decomp]
+    end
+  end
+  return f
+end
+
+function extend(f::BlowUpSequence, g:: BlowUpSequence)
+  for g_i in morphisms(g)
+    add_map!(f,g_i)
+  end
+  return f
+end
+
+function update_dont_meet_pts!(f::Union{BlowUpSequence, MixedBlowUpSequence}, I::AbsIdealSheaf)
+  dim(I) == 0 || return f
+  patches = Oscar.patches(default_covering(scheme(I)))
+  if !isdefined(f,:dont_meet)
+    dont_meet = Vector{Tuple{Int,Int}}()
+  else
+    dont_meet = f.dont_meet
+  end
```

Comment:
`dont_meet` might not be visible outside this `if`-block. Better do
```suggestion
  dont_meet = isdefined(f, :dont_meet) ? f.dont_meet : Vector{Tuple{Int, Int}}()
```

Reply:
I believe this is no concern since `if` blocks don't have their own soft scope (though `for` and `while` do). 
(https://docs.julialang.org/en/v1/manual/variables-and-scoping/)

Reply:
done